pat/ReferenceTestV2/reference_provider_v2.py

```diff
@@ -12,19 +12,13 @@
 from __future__ import annotations
```

Comment:
in this file, maybe add in line 323 
`cast(descriptorcontainers.AlertConditionDescriptorContainer, ... )`